src/webgl/p5.RendererGL.js

```diff
@@ -1015,8 +1015,18 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
           filterShaderVert,
           filterShaderFrags[operation]
         );
+
+        // two-pass blur filter needs another shader attached to main
+        if (operation === constants.BLUR) {
```

Comment:
What if someone is trying to use the accelerated filters from p2d mode. Won't this cause an error?